stdpopsim/catalog/SusScr/species.py

```diff
@@ -0,0 +1,150 @@
+import stdpopsim
+
+from . import genome_data
+
+_ZhangEtAl = stdpopsim.Citation(
+    doi="https://doi.org/10.1016/j.gpb.2022.02.001",
+    year=2022,
+    author="Mingpeng Zhang et al.",
+    reasons={
+        stdpopsim.CiteReason.MUT_RATE,
+        stdpopsim.CiteReason.GEN_TIME,
+        stdpopsim.CiteReason.POP_SIZE,
+    },
+)
+_JohnssonEtAl = stdpopsim.Citation(
+    doi="https://doi.org/10.1186/s12711-021-00643-0",
+    year=2021,
+    author="Martin Johnsson et al.",
+    reasons={stdpopsim.CiteReason.REC_RATE},
+)
+# [The following are notes for implementers and should be deleted
```

Comment:
@AprilYUZhang cite the paper here in the comments and point to page/figure/table for there estimates